Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -36,52 +36,18 @@ public class NavigationCameraStateTransition: CameraStateTransition {
     
     public func transitionToFollowing(_ cameraOptions: CameraOptions, completion: @escaping (() -> Void)) {
         guard let mapView = mapView,
-              let zoom = cameraOptions.zoom,
-              let location = cameraOptions.center,
-              let bearing = cameraOptions.bearing,
-              let pitch = cameraOptions.pitch,
-              let padding = cameraOptions.padding,
-              let anchor = cameraOptions.anchor else {
+              let zoom = cameraOptions.zoom else {
             completion()
             return
         }
-        
-        if transitionDestinationIsOffScreen(location, edgeInsets: padding) {
-            let screenCenterPoint = self.screenCenterPoint(0.0, bounds: mapView.bounds, edgeInsets: padding)
-            let lineString = LineString([mapView.cameraState.center, location])
-            var camera = mapView.mapboxMap.camera(for: .lineString(lineString), padding: .zero, bearing: nil, pitch: nil)
-            camera.bearing = mapView.cameraState.bearing
-            camera.pitch = 0.0
-            if let midPointZoom = camera.zoom {
```

Comment:
Two-step animation for cases when destination is off-screen was replaced with simpler one-step animation.